network/execute_probe.js

```diff
@@ -49,6 +49,7 @@ const tokensAsMillitokens = tok => (BigInt(tok) * BigInt(1e3)).toString();
       icons: [<Icon String>]
       public_key: <Public Key Hex String>
     }]
+    [strict_max_fee]: < Strict Maximum Fee Tokens Number>
```

Comment:
```suggestion
    [strict_max_fee]: <Strict Maximum Fee Tokens Number>
```

Reply:
These lines won't be necessary anymore I guess? Since we are not passing a strict_max_fee anymore and only passing a max_fee which could become strict with `--avoid-high-fee-routes`

Reply:
ok yeah if there is no reason to have two fees